server/athenian/api/precompute/accounts.py

```diff
@@ -203,16 +203,15 @@ async def precompute_reposet(
         context.slack,
     )
     try:
-        prefixer, bots, new_items = await gather(
+        prefixer, bots = await gather(
             Prefixer.load(meta_ids, mdb, cache),
             fetch_bots(reposet.owner_id, meta_ids, mdb, sdb, None),
-            refresh_repository_names(reposet.owner_id, meta_ids, sdb, mdb, None),
         )
     except Exception as e:
         log.error("prolog %d: %s: %s", reposet.owner_id, type(e).__name__, e)
         sentry_sdk.capture_exception(e)
         return
-    reposet.items = new_items
+    reposet.items = prefixer.dereference_repositories(reposet_items_to_refs(reposet.items))
```

Comment:
expression has a different type than `reposet.items` and if I'm not wrong `reposet.items` is  not saved after.
Could we just use a separate variable ?

Reply:
Sure, this is just some legacy of how it used to work.